API.md

```diff
@@ -22,3 +22,8 @@ This page documents the SDX Gateway API endpoints. Apart from the Service Inform
 **Required parameters**: `caseId` as the ID of the case the response receipt is for.
 
 An `HTTP 201 Created` status code is returned if the response receipt creation was a success. An `HTTP 400 Bad Request` is returned if the required parameter is missing.
+
+## Acknowledge File
+* `POST /paperquestionnairereceipts`
+
+An `HTTP 201 Created` status code is returned if the file was read.
```

Comment:
```suggestion
A `HTTP 201 Created` status code is returned if the file was read.
```

Reply:
Using "An" rather than "A" is grammatically correct here, because "HTTP" begins with an "/eych/" sound. Saying "/Ah eych-tee-tee-pee/" sounds awkward. It's like how you say "an hour", not "a hour".

https://www.writersdigest.com/online-editor/a-before-consonants-and-an-before-vowels-is-not-the-rule

Reply:
Well you lean something new everyday